### PR TITLE
Skip Chainer tests for CuPy>=8

### PR DESCRIPTION
**Pull request by niboshi**
Skip `chainer-*` test matrices in Jenkins `cupy_pr` test job, if cupy>=8.

### CODE REVIEW

**Comment by emcastillo** [comment]
Jenkins, test this please

File: run_test.py [hunk 1]
```diff
@@ -35,6 +35,11 @@
 ]
 
 
+def _get_job_name():
+    # Returns Jenkins job name
+    return os.environ['JOB_NAME']
```
Comment (toslunar) [inline run_test.py]: How about
```suggestion
    return os.environ.get('JOB_NAME')
```
to support manual tests?

Comment (toslunar) [reply run_test.py]: Sorry.  `argconfig.py` uses `os.getenv` instead.
```suggestion
    return os.getenv('JOB_NAME')
```

Comment (niboshi) [reply run_test.py]: Fixed in 2fabab1d72cc0c048fcb0159152a6e0f77f2d755. Thanks.

**Comment by emcastillo** [comment]
Jenkins, test this please

**Pull request merged**